ci/cleanup.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+UNREAL_PATH="${1:-$(pwd)/../UnrealEngine}"
+BUILD_PROJECT=${2:-NetworkTestProject}
+
+PROJECT_ABSOLUTE_PATH="$(pwd)/../${BUILD_PROJECT}"
+GDK_IN_TEST_REPO="${PROJECT_ABSOLUTE_PATH}/Game/Plugins/UnrealGDK"
+
+# Workaround for UNR-2156 and UNR-2076, where spatiald / runtime processes sometimes never close, or where runtimes are orphaned
+# Clean up any spatiald and java (i.e. runtime) processes that may not have been shut down
+#spatial service stop
```

Comment:
commented out?

Reply:
yes, for testing purposes. will come back as soon as I'm sure it works correctly